### ISSUE TITLE
Uberbar Not accessible via VoiceOver Rotor (Landmarks)

### ISSUE THREAD

**Issue by jpdevries** [modxcms/revolution#13436]
### Summary
The uber bar is not accessible via assistive tech landmarks such as the VoiceOver rotor.

![](http://j4p.us/0J1G2p2W3u0O/Screen%20Shot%202017-04-23%20at%203.03.13%20PM.png)

### Step to reproduce
Using VoiceOver, open landmarks. Notice the search component is not present.

### Observed behavior
Can't access uberbar via the landmarks rotor.

### Expected behavior
Should be able to access uberbar via the landmarks rotor.

### Environment
MODX 2.x

### Solution
I tried to send a PR for this but the feeling between ExtJS and I is mutually. Here's the markup we have now:

```html 
<li id="modx-manager-search">
  <div class="x-form-field-wrap x-form-field-trigger-wrap" id="ext-gen274" style="width: 207px;">
    <input type="text" size="24" autocomplete="off" id="modx-uberbar" name="modx-uberbar" class="x-form-text x-form-field x-form-empty-field" style="width: 169px;">
    <div class="x-form-trigger x-form-arrow-trigger" id="ext-gen275"></div>
  </div>
</li>
```

Here's a fix:
```html 
<li id="modx-manager-search" role="search">
  <div class="x-form-field-wrap x-form-field-trigger-wrap" id="ext-gen274" style="width: 207px;">
    <input aria-label="Search" type="text" size="24" autocomplete="off" id="modx-uberbar" name="modx-uberbar" class="x-form-text x-form-field x-form-empty-field" style="width: 169px;">
    <button type="submit" aria-label="Go" class="x-form-trigger x-form-arrow-trigger" id="ext-gen275" style="margin:0"></button>
  </div>
</li>
```

So to recap
 - [x] `role="search"` on `#modx-manager-search` (or wrap the input in a actual `<form role="search">`)
 - [x] `aria-label` on the Input (not sure where the Search "label" is coming from now)
 - [x] Change the submit "button" to an actual button
 - [x] Give the submit button `aria-label` and `type="submit"`
 - [ ] Styling to remove `margin` form the submit button

* * * *
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by rtripault** [comment]
Your turn again now :)

**Comment by Ibochkarev** [comment]
Improved in #13436
